ui/components/component-library/avatar-token/avatar-token.stories.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import {
+  COLORS,
+  SIZES,
+  DISPLAY,
+  ALIGN_ITEMS,
+} from '../../../helpers/constants/design-system';
+
+import Box from '../../ui/box/box';
+
+import README from './README.mdx';
+import { AvatarToken } from './avatar-token';
+
+export default {
+  title: 'Components/ComponentLibrary/AvatarToken',
+  id: __filename,
+  component: AvatarToken,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    size: {
+      control: 'select',
+      options: Object.values(SIZES),
+    },
+    tokenName: {
+      control: 'text',
+    },
+    tokenImageUrl: {
+      control: 'text',
+    },
+    showHalo: {
+      control: 'boolean',
+    },
+  },
+  args: {
+    tokenName: 'ast',
+    tokenImageUrl: './AST.png',
+    size: SIZES.MD,
+    showHalo: false,
+  },
+};
+
+const Template = (args) => {
+  return <AvatarToken {...args} />;
+};
+
+export const DefaultStory = Template.bind({});
+DefaultStory.storyName = 'Default';
+
+export const Size = (args) => (
+  <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.BASELINE} gap={1}>
+    <AvatarToken {...args} size={SIZES.XS} />
+    <AvatarToken {...args} size={SIZES.SM} />
+    <AvatarToken {...args} size={SIZES.MD} />
+    <AvatarToken {...args} size={SIZES.LG} />
+    <AvatarToken {...args} size={SIZES.XL} />
+  </Box>
+);
+
+export const tokenName = Template.bind({});
+tokenName.args = {
+  tokenImageUrl: '',
+};
+
+export const tokenImageUrl = Template.bind({});
+
+export const showHalo = Template.bind({});
+showHalo.args = {
+  showHalo: true,
+};
+
+export const BackgroundAndBorderColor = (args) => (
```

Comment:
Update this so it matches title in MDX doc
```suggestion
export const ColorBackgroundColorAndBorderColor = (args) => (
```